tests/modules/suite2p_registration/test_suite2p_registration_pipeline.py

```diff
@@ -97,8 +97,11 @@ def test_suite2p_motion_correction(
                                    prefix='output_',
                                    suffix='.json')[1]
 
+    str_tmpdir = str(pathlib.Path(tmpdir).resolve().absolute())
```

Comment:
I think these days it is preferred to use the `tmp_path` fixture (and do `tmp_path.mkdir()`) instead of using the `tmpdir` fixture which provides only a `pathlike str`.

Reply:
Thanks for the heads up. I've incorporated the change and will try to remember to use this in the future.